packages/protocol/contracts/L1/libs/LibProposing.sol

```diff
@@ -148,19 +150,33 @@ library LibProposing {
             revert L1_TOO_MANY_BLOCKS();
         }
 
+        address permittedProposer = _resolver.resolve(LibStrings.B_BLOCK_PROPOSER, true);
```

Comment:
Should this `permittedProposer` come from `PreconfTaskManager `instead of an address from `AddressManager`?

Reply:
permittedProposer is PreconfTaskManager in this case.